ee/vellum_ee/workflows/display/nodes/types.py

```diff
@@ -8,6 +11,14 @@ class NodeOutputDisplay:
     name: str
 
 
+@dataclass
+class NodeInputDisplay:
+    id: UUID
+    name: str
+    type: Optional[str] = None
+    value: Optional[WorkflowValueDescriptor] = None
```

Comment:
[blocking] I'm not following why we need this? there's no new data that this is tracking afaict. I have a sync with a customer from 4-430p, let's sync after